tests/components/rest/test_init.py

```diff
@@ -418,3 +418,21 @@ async def test_empty_config(hass: HomeAssistant) -> None:
         {DOMAIN: {}},
     )
     assert_setup_component(0, DOMAIN)
+
+
+async def test_config_schema_via_packages(hass: HomeAssistant) -> None:
+    """Test configuration via packages."""
+    packages = {
+        "pack_dict": {"rest": {}},
+        "pack_11": {"rest": {"resource": "http://url1"}},
+        "pack_list": {"rest": [{"resource": "http://url2"}]},
+    }
+    config = {hass_config.CONF_CORE: {hass_config.CONF_PACKAGES: packages}}
+    with patch("homeassistant.config._LOGGER.error") as logerr:
+        await hass_config.merge_packages_config(hass, config, packages)
```

Comment:
Can we use `caplog` here instead?

Reply:
I copied an existing test from config
I usually use caplog for positive assertions.

Reply:
Maybe I just remove logger checks?

Reply:
> Maybe I just remove logger checks?

That works.   Patching logger is kinda meh

Reply:
> That works. Patching logger is kinda meh

Done